### PR TITLE
Add aria roles

### PR DESCRIPTION
**Pull request by ebrillhart**
This adds the necessary aria roles to victory-pie and adds the new title and desc props for users to provide more information to screen readers. Title defaults to the type of chart and desc defaults to an empty string. @paulathevalley @boygirl

### CODE REVIEW

**Comment by ebrillhart** [comment]
@boygirl this should also be good I think

File: src/components/victory-pie.js [hunk 1]
```diff
@@ -267,7 +268,22 @@ export default class VictoryPie extends React.Component {
...
+     * The containerComponent prop takes an entire component which will be used to surround
+     * a nested chart component unless the standalone prop is set to true.
+     * The container component will be provided the following properties calculated by
+     * the chart component it surrounds: height, width, a child component (the chart itself) and
+     * style. Props that are not provided by the child chart component include title and desc,
+     * both of which are intended to add accessibility to Victory components. The more descriptive
+     * these props are, the more accessible your data will be for people using screen readers.
+     * Any of these props may be overridden by passing in props to the supplied component,
+     * or modified or ignored within the custom component itself. If a dataComponent is
+     * not provided, VictoryStack will use the default VictoryContainer component.
```
Comment (boygirl) [inline src/components/victory-pie.js]: should reference VictoryPie not VictoryStack

File: src/components/victory-pie.js [hunk 2]
```diff
@@ -267,7 +268,22 @@ export default class VictoryPie extends React.Component {
       CustomPropTypes.allOfType([CustomPropTypes.integer, CustomPropTypes.nonNegative]),
       PropTypes.string,
       PropTypes.arrayOf(PropTypes.string)
-    ])
+    ]),
+    /**
+     * The containerComponent prop takes an entire component which will be used to surround
```
Comment (boygirl) [inline src/components/victory-pie.js]: this wording is a little clunky... maybe "The containerComponent prop takes an entire component which will be used to create a container element for standalone charts. The new element created from the passed containerComponent will be provided with ... props... provided by VictoryPie. Something about title and desc... caveats about overridden props... "

**Comment by boygirl** [comment]
@ebrillhart looks good to me pending some clean up of the prop description

**Comment by ebrillhart** [comment]
closes https://github.com/FormidableLabs/victory/issues/263

**Pull request merged**